### DISCUSSION TITLE
How to yank from Special Register to System Clipboard?

### DISCUSSION THREAD

**Discussion by Sammo98** [helix-editor/helix#8012] [Q&A]
Hi all,

I've built off of current master which has this merged in:

https://github.com/helix-editor/helix/pull/6985

This enables the use of `"%p` which will paste the path of the current buffer, is there a way to yank from this register into the system register?

**Comment by the-mikedavis** [comment]
You can paste from the source register and yank to the target register: `"%p"*d` to copy the filename register's value to the system clipboard. I think it could be useful to have a command that copies values between registers though since pasting and deleting adds an undo checkpoint.

**Reply by zhaob1n** [reply]
> I think it could be useful to have a command that copies values between registers though since pasting and deleting adds an undo checkpoint.

I think so. Is it implemented now? Or is there a better way to copy values between registers?

**Reply by zhaob1n** [reply]
Never mind, I find it's already implemented.
